README.md

```diff
@@ -1,2 +1,21 @@
-# hangman
-hangman
+# Hangman
+Em Ay Devs members tend to spend a lot of time playing hangman—usually by drawing things poorly—so we decided to create our own version of the game.
```

Comment:
✏️ Shouldn't it be spelled `Em-Ay-Devs`?

Reply:
- It is only written as Em-Ay-Devs everywhere else because of the we can't use spaces. We don't have that limitation here.

- That's not a hyphen.

Reply:
I don't really agree with your usage of hyphens here, but it's not a big deal.

Reply:
Edit: Never mind, I was reading this incorrectly.